src/System.Diagnostics.Tools/tests/project.json

```diff
@@ -13,7 +13,8 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00807-03"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.3": {},
+    "netstandard1.7": {}
   },
   "supports": {
     "coreFx.Test.netcore50": {},
```

Comment:
you will need to add corefx.Test.netcoreapp1.1 to get your netstandard17 test configuration to run.